docs/UserGuide.md

```diff
@@ -183,152 +193,215 @@ Adds an event with specific starting and ending date or time to TaskTracker.
   date.
 
   The default year of `dd/mm` is the current year and the default month for
-  `dd` is current month.
+  `dd` is the current month.
 
-* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCD` which
-  each letter represents a number ranges from 0 to 9. `AB` represent the hour
-  in 24 hour system, while `CD` represent the minute. Letter `A` can be omitted
-  if it is `0`.
+* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCDam` or `ABCDpm`
+  where each letter represents a digit. `AB` represent the hour
+  in 12 hour system, while `CD` represent the minute. Digit `A` can only be omitted
+  if it is `0`, and 'CD' can be only omitted (together) when both of them are `0`.
 
-* The default starting date will be the same as the real date on user's
-  computer.
+* The default starting date will be the current date.
 
 * The default ending date will be the same as the starting date.
 
-* The default starting time will be `0000`.
+* The default starting time will be `0000am` of starting date.
```

Comment:
> 12am is the start of a day or the end? For me, I think 12am should be the start, but I think it will be more reasonable to set the start as 0.

It's not really what is "technically correct". If we want to be technically correct, we should not be using 12am or 0am at all since they are not valid 12-hour clock timings.

Rather, it is about convention, and that means looking at what most apps already do. I think, for example, that the Windows clock uses 12am to refer to midnight (the start of the day).